apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/store.js

```diff
@@ -0,0 +1,35 @@
+import { createRegistrySelector, registerStore } from '@wordpress/data';
+
+const DEFAULT_STATE = {
+	isModalVisible: true,
+};
+
+registerStore( 'automattic/wpcom-global-styles', {
+	reducer: ( state = DEFAULT_STATE, action ) => {
+		switch ( action.type ) {
+			case 'DISMISS_MODAL':
+				return {
+					...state,
+					isModalVisible: false,
+				};
+		}
+
+		return state;
+	},
+
+	actions: {
+		dismissModal: () => ( {
+			type: 'DISMISS_MODAL',
+		} ),
+	},
+
+	selectors: {
+		isModalVisible: createRegistrySelector( ( select ) => ( state ) => {
```

Comment:
Nitpick, we can remove the parenthesis of these arguments since these arrow functions only need one argument.

Reply:
https://github.com/Automattic/wp-calypso/pull/68530#discussion_r986549238